templates/lib/metatype.cpp

```diff
@@ -106,12 +110,89 @@ static QVariant doQobjectLookUp(const QObject *const object,
   return object->property(property.toUtf8().constData());
 }
 
+static QVariant doVariantListLookUp(const QVariantList &list,
```

Comment:
What I meant was here (and on the hash lookup), change the type to a QJsonArray and return the QJsonValue::toVariant() of it